### DISCUSSION TITLE
Cant use the custom power plans after the updates

### DISCUSSION THREAD

**Discussion by Yaksi** [seerge/g-helper#1453] [Q&A]
It was working after a while I noticed that it keeps stuck at being balanced plan on any power mode and I did close the g helper and change the config.json, but it keeps making a new one.

{
  "performance_mode": 1,
  "performance_0": 2,
  "gpu_mode": 1,
  "internal_display": "",
  "screen_max": 60,
  "frequency": 60,
  "overdrive": 0,
  "performance_1": 1,
  "charge_limit": 80,
  "charge_full": 0
  "scheme_0": "381b4222-f694-41f0-9685-ff5bb260df2e",
  "scheme_1": "c29f0940-7c22-4fbc-8c6b-7398d95a8642",
  "scheme_2": "2c7d7d02-fd7a-4902-a86a-2470336a0fcb"
}
 

this is how I wrote it normally it was working

**Comment by seerge** [comment]
@Yaksi you are literally missing a comma after

```
"charge_full": 0
```

So it makes JSON structure broken, and after failed attempt to read it - it recreates it.

Add a comma

https://jsonlint.com/

**Comment by Yaksi** [comment]
My bad how did I miss it anyway, thanks for helping and doing it in such a quick manner.
